workspaces/odo/plugins/odo-module-devfile-field-extension/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@backstage-community/plugin-odo-module-devfile-field-extension",
+  "version": "0.20.0",
```

Comment:
```suggestion
  "version": "0.0.0",
```

As this is new to this repo, let's start at `0.0.0`, it will bump when it get's merged and release to `0.1.0`

Reply:
Sure, if there's prior art for this then I'm cool with that 👍